src/Core/src/Handlers/Entry/EntryHandler.Android.cs

```diff
@@ -113,6 +115,11 @@ public static void MapPlaceholder(EntryHandler handler, IEntry entry)
 			handler.NativeView?.UpdatePlaceholder(entry);
 		}
 
+		public static void MapPlaceholderColor(EntryHandler handler, IEntry entry) 
+		{
+			handler.NativeView?.UpdatePlaceholderColor(entry, DefaultPlaceholderColors);
```

Comment:
Can we add an `UpdatePlaceholderColor` on iOS as well?  I realize iOS has to call `UpdatePlaceholder` and update both values but it's nice having symmetrical APIs for the platforms